queries.yaml

```diff
@@ -77,7 +92,6 @@ layers:
     area-inclusion-threshold: 1
   roads:
     template: roads.jinja2
-    start_zoom: 5
```

Comment:
Looks like we source the Natural Earth scale_rank for min_zoom directly, and that starts at 3:

```
    min_zoom: { col: scalerank }
```

So this change would introduce roads at zoom 3 unless paired with a clamp over in the roads layer YAML?